.azure-pipelines/windows/basic-tests.yml

```diff
@@ -0,0 +1,25 @@
+#
+# Azure Pipelines configuration for building and testing react on Windows.
+#
+
+steps:
+  - script: yarn build
+    displayName: 'Build'
+
+  - script: set JEST_JUNIT_OUTPUT=.azure-pipelines/test-results/test-basic-windows.xml&&yarn test
```

Comment:
I've been setting the `env` property of `script` when i need a variable to exist only for one step:

```yaml
script: yarn test
env:
  JEST_JUNIT_OUTPUT: '.azure-pipelines/test-results/test-basic-windows.xml'
```